feature/policy_forwarding/encapsulation/otg_tests/staticgueencap_and_bgp_path_selection/README.md

```diff
@@ -0,0 +1,709 @@
+    *   The DUT should accurately stream data regarding the number of
+        packets/bytes encapsulated
+    *   Unencapsulated flows from ATE2 to `ATE1_Port1` must have 100% success
+        (Zero loss), routing via the IBGP peering between `$ATE2_IBGP.v[46]` and
+        `$DUT_lo0.v[46]`
+    *   Post-test health checks should be performed and compared against the
+        baseline. Verify the absence of drops or core dumps. If any, the test
+        Must fail
+
+**RT-3.52.3: AF1-AF4 Traffic Migrated to DUT --> ATE2_Port3**
+
+Follow the same steps as in RT-3.52.2 and gradually move one Traffic class at a
+time in the following order. Note changes in RT-3.52.5 and RT-3.52.6
+
+*   **RT-3.52.3: Migrate AF1 Flows:** Migrate routing of AF1 flows from
+    `DUT_Port2` --> `ATE2_Port1` to `DUT_Port4` --> `ATE2_Port3`.
+
+    *   BE1 and AF1 are now migrated
+
+    *   **RT-3.52.4: Migrate AF2 Flows:** Migrate routing of AF2 flows from
+        `DUT_Port2` --> `ATE2_Port1` to `DUT_Port4` --> `ATE2_Port3`.
+
+        *   BE1-AF2 are now migrated
+
+    *   **RT-3.52.5: Migrate AF3 Flows:** Migrate routing of AF3 flows from
+        `DUT_Port2` --> `ATE2_Port1` to `DUT_Port4` --> `ATE2_Port3`.
+
+        *   `$ATE2_C.IBGP.v6` will advertise `$ATE2_INTERNAL9.v4/24` and
+            `$ATE2_INTERNAL9.v6/64` with next-hop as `$ATE2_PPNH2.v6/128`.
+            Traffic towards `$ATE2_INTERNAL9.v[46]/24` will have tunnel
+            destination `$ATE2_INTERNAL_TE10.v4/32`.
+        *   BE1-AF3 are now migrated
+
+    *   **RT-3.52.6: Migrate AF4 Flows:** Migrate routing of AF4 flows from
+        `DUT_Port2` --> `ATE2_Port1` to `DUT_Port4` --> `ATE2_Port3`.
+
+        *   `$ATE2_C.IBGP.v6` will advertise `$ATE2_INTERNAL10.v4/24` and
+            `$ATE2_INTERNAL10.v6/64` with next-hop as `$ATE2_PPNH2.v6/128`
+            Traffic towards `$ATE2_INTERNAL10.v[46]` will have tunnel
+            destination `$ATE2_INTERNAL_TE10.v4/32`.
+        *   BE1-AF4 are now migrated
+
+*   **Expectations:**
+
+    *   Same as RT-3.52.2 for each traffic class migrated except that AF3 and
+        AF4 traffic are tunneled towards `$ATE2_INTERNAL_TE10.v4/32`.
+
+**RT-3.52.7: DUT as a GUE Decap Node**
+
+*   **Situation:**
+
+    *   The test begins from the state established in RT-3.52.3, where all
+        traffic from ATE1 to ATE2 is encapsulated by the DUT and routed via
+        DUT_Port4 --> ATE2_Port3 path.
+
+*   **Test Steps:**
+
+    *   Perform all previously defined health checks as a baseline.
+    *   Stop Flow-Set #5 and start Flow-Set #4, resulting in active flows for
+        Flow-Set #1 through Flow-Set #4.
+
+*   **Expectations:**
+
+    *   Traffic from ATE1 to ATE2 should be GUE encapsulated with tunnel
+        destinations `$ATE2_INTERNAL_TE11.v4/32` and `$ATE2_INTERNAL_TE10.v4/32`
+        and routed out `$DUT_Port4<>$ATE2_Port3`.
+
+    *   The DUT should provide accurate streaming data for the number of
```

Comment:
specify OC paths needed for this (here and in the OC Paths at the yaml stanza at end of this README)

Reply:
Done